week-2/J-array-get-set/exercise.js

```diff
@@ -5,11 +5,11 @@
 */
 
 function first(arr) {
-  return; // complete this statement
+  return arr [0]; // complete this statement
 }
 
 function last(arr) {
-  return; // complete this statement
+  return arr [2] ; // complete this statement
```

Comment:
What if the parameter `arr` passed to this function has 4 items? Would this still work?

Try to re-write this without hardcoding the last value.